module/mempool/stdmap/backDataHeapBenchmark_test.go

```diff
@@ -108,7 +108,7 @@ func testAddEntities(t testing.TB, limit uint, b *stdmap.Backend, entities []*un
 // it compliant to be used as BackData component in mempool.Backend. Note that
 // this is used only as an experimental baseline, and so it's not exported for
 // production.
-type baselineLRU struct {
+type baselineLRU[K comparable, V any] struct {
 	c     *lru.Cache // used to incorporate an LRU cache
```

Comment:
```suggestion
	c     *lru.Cache[K, V] // used to incorporate an LRU cache
```

We can use `lru.Cache[K, V]` here if we import `lru "github.com/hashicorp/golang-lru/v2"` instead of `lru "github.com/hashicorp/golang-lru"`

This will make comparison more precise